configurations/html5.template.php

```diff
@@ -28,6 +28,10 @@
 $wgKalturaCDNUrl = wgGetUrl('cdn_host');
 // Default Stats URL
 $wgKalturaStatsServiceUrl = wgGetUrl('stats_host');
+// Default Live Stats URL
+$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host');
+// Default Kaltura Analytics URL
+$wgKalturaAnalyticsServiceUrl = wgGetUrl('analytics_host');
 
 // SSL host names
 if( $wgHTTPProtocol == 'https' ){
```

Comment:
see the difference:
$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host_https');
is not 
$wgKalturaStatsServiceUrl = wgGetUrl('stats_host_https');